server/db/db.js

```diff
@@ -694,6 +694,33 @@ async function getLocationsFromClientId(clientId, pgClient) {
   }
 }
 
+async function numberOfStillnessAlertsIn24Hours(locationid, pgClient) {
+  const endDate = new Date()
+  const startDate = new Date(endDate.getTime() - 24 * 60 * 60 * 1000)
```

Comment:
As we talked about on our call but that I'm mention here again so I don't forget to check for it again later... I would use the postgres command `NOW()` and the keyword `INTERVAL` to do this in the query itself because time is tricky and the server running the code may have a different time than the server running the DB.